server/routerlicious/packages/lambdas/src/foreman/lambda.ts

```diff
@@ -99,14 +100,13 @@ export class ForemanLambda implements core.IPartitionLambda {
                         type: "tasks:start",
                     },
                 );
-                this.context.log?.info(
-                    `Request to ${queueName}: ${clientId}:${JSON.stringify(tasks)}`,
-                    {
-                        messageMetaData: {
-                            documentId: docId,
-                            tenantId,
-                        },
```

Comment:
Let's also keep the old way of logging while we validate the new pipeline. We will remove the old codepath later.